test/mar.jl

```diff
@@ -50,12 +49,11 @@ end
     for (problem_set, optimizer) in problem_sets
         @testset "$(problem_set) problem set" begin
             # Capture the problem names that belong to the current problem set
-            problem_names = get_problem_names(problem_set, "MAR")
+            problem_names = TensorInference.get_problem_names("uai2014", problem_set, "MAR")
```

Comment:
I think this function can more intuitive. Like
```julia
TensorInference.instances_in_artifact("uai2014")
# output is a dictionary of problem_set -> task -> problem_name
```

Reply:
NOTE: Since I have a related update related to #49 , I am implementing this change. So, l will merge this PR directly to allow me work on another PR.

Reply:
Sure! No problem.